spec/remote_io_spec.rb

```diff
@@ -7,7 +7,7 @@
     rio = described_class.new('https://images.invalid/img.jpg')
 
     fake_resp = double(headers: {'Content-Range' => '10-109/2577'}, status: 206, body: 'This is the response')
-    expect(Faraday).to receive(:get).with('https://images.invalid/img.jpg', nil, range: 'bytes=10-109').and_return(fake_resp)
+    expect_any_instance_of(Faraday::Connection).to receive(:get).with('https://images.invalid/img.jpg', nil, range: 'bytes=10-109').and_return(fake_resp)
```

Comment:
`expect_any_instance_of` is deprecated for some time now and it is advised against using it. Would it be a better idea to create a stub for `Faraday::Connection` and `expect` on it (similar to what you did below)?

Reply:
It's weird because RSpec doesn't warn about that in the output, but it is indeed deprecated.

I changed all of them

Thank you!